velox/exec/fuzzer/CacheFuzzer.cpp

```diff
@@ -60,21 +60,6 @@ DEFINE_int64(memory_cache_bytes, 32 << 20, "Memory cache size in bytes.");
 
 DEFINE_uint64(ssd_cache_bytes, 128 << 20, "Ssd cache size in bytes.");
 
-DEFINE_int32(num_ssd_cache_shards, 4, "Number of SSD cache shards.");
```

Comment:
Can we randomize if those values are special such as -1 so it is easier for us to reproduce? Correspondingly, it is better to print out those settings at the start of the test? Thanks!